docs/source/deployment_guide_overview.md

```diff
@@ -0,0 +1,173 @@
+
+### Managing your infrastructure
+
+The exact methods and tools you use to manage your backend will depend on the backend you choose. However, here are some considerations worth noting.
+
+* Using secret objects to securely store important configuration files in your cluster. For information about Kubernetes secrets, check out [Kubernetes secrets](https://kubernetes.io/docs/concepts/configuration/secret/).
+* Cluster considerations and node sizing. In step 2 above, we discussed a general outline for how to think about the sizings of nodes. Your use case, as well as a robust period of development, is the only way you will truly know how how large your peers, ordering nodes, and CAs will need to be.
+* How you choose to mount your volumes. It is a best practice to mount the volumes relevant to your nodes external to the place where your nodes are deployed. This will allow you to reference these volumes later on (for example, to deploy nodes with virtually identical configurations) without having to enter the container where the node has been deployed.
+* How you will monitor your resources. It is critical that you establish a strategy and method for monitoring the resources used by your individual nodes and the resources deployed to your cluster generally. As you join your peers to more channels, you will need likely need to increase its CPU and memory allocation. Similarly, you will need to make sure you have enough storage space for your state database and blockchain.
+
+## Step three: Set up your CAs
+
+The first component that must be deployed in a Fabric is a CA. This is because the configuration of a component (such as a peer, an ordering node, or the MSP representing an organization) must include at least one user identity that is authorized to operate it before it can be deployed. While it is not necessary to use the Fabric CA, the Fabric CA also creates MSP structures that are needed for components and organizations to be properly defined. If a user choose to use a CA other than the Fabric CA, they will have to create the MSP folders themselves.
+
+While all of the certificates associated with an organization can be created by a single "root" CA (that is, a CA that is its own root of trust), for added security organizations can decide to use "intermediate" CAs (for example, two intermediate CAs, one for org enrollment and another for TLS) whose certificates are created by a root CA (or another intermediate CA that eventually leads back to a root CA). Because a compromise in the root CA leads to a collapse for its entire trust domain (the certs for the admins, nodes, and any CAs it has generated certificates for), intermediate CAs are a useful way to limit the exposure of the root CA. Whether you choose to use intermediate CAs will depend on the needs of your use case. They are not mandatory.
+
```

Comment:
As such, it is also possible to configure Lightweight Directory Access Protocol (LDAP) to manage identities on a Fabric network for those enterprises that already have this implementation and do not want to add a layer of identity management to their existing infrastructure.